example/package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
+    "release-ios": "react-native run-ios --configuration Release",
+    "release-android": "react-native run-android --variant=Release",
```

Comment:
this naming is a bit confusing to me because it sounds like this script actually releases the SDK, but it just runs the app in the release config, right?

Maybe we could rename to `ios-release-mode` and `android-release-mode`?